pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -58,6 +59,15 @@ const (
 	Unknown        = "unknown"
 )
 
+// CPU Pinning Policies
+const (
+	None           = "none"
+	Manual         = "manual"
+	ResizeAndPin   = "resize_and_pin_numa"
+	Dedicated      = "dedicated"
+	IsolateThreads = "isolate_threads"
```

Comment:
For manual, resize and isolated - a PR for validation should be added where a warning will be shown that it won't be respected.
As for the dedicated - we may add a warning as well about the requirement enabling `CPU manager`. - that's an open question.

Reply:
If the migration blocks without CPU manager and we know that it is not enabled on destination then I would go with an error and not just warning.

Reply:
OK, one way to do it in my opinion is listing the hosts and check for the label `cpumanager=true`. i don't know if that's enough. also, i'm unsure if it's possible to do on validation before starting migration or only blocking during migration.

Reply:
I think you're talking about two different things - Tomas talks about the migration being block, as if the VM is rejected in this case, you Liran talks about the inability to schedule the VM. is that right?

Reply:
No. Because you can't schedule the VM you can't proceed in the migration flow. Therefore, it actually blocks the migration.

Reply:
so you say that the vm starts during the migration? the VM was down at the end of the migrations I did and I don't remember seeing it up..

Reply:
yes, it's the `ScheduleVM` step. This triggers the copy disks. On `CopyDisks` we check the progress.

Reply:
ok, in that case you both were talking about the same thing indeed - but that makes it simpler. if we don't check for other things that would "ensure" the vm can be scheduled on any of the hosts, I don't see why doing it here - forklift should handle that in a more generic way (not on a per-property basis) I guess

Reply:
> For manual, resize and isolated - a PR for validation should be added where a warning will be shown that it won't be respected. 

So this PR is only supposed to set the "dedicated" flag, if that is what the source VM wants, and just ignore all other values, right?

Reply:
yes

Reply:
Ok, so if we have same problem with other parameters (mem, cpu,..?) and we don't error there then yes, let's keep the behavior consistent.

Reply:
based on the offline discussion, we shouldn't check `dedicated` as a special case. CNV cluster should have node with `cpumanger` enabled, and it's the same as broader problem of not having enough resources(like memory and so on).
we can add a warning/info about it, but that also open to discussion.
the relevant validation will be executed in another PR to forklift-validation.